src/mlpack/methods/gmm/gmm_impl.hpp

```diff
@@ -3,6 +3,7 @@
  * @author Parikshit Ram (pram@cc.gatech.edu)
  * @author Ryan Curtin
  * @author Michael Fox
+ * @author Rohan Raj
```

Comment:
Please add your name only to the files you have changed / made significant code-based contributions to.

Reply:
This is still unchanged.

Reply:
Apologies for skipping this. Thanks